### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by axos88-da**
update cookbook version in readme.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -48,7 +48,7 @@ There are no hard coupled dependencies. However, there is a loose dependency on
 Place a dependency on the mysql cookbook in your cookbook's metadata.rb
 
 ```ruby
-depends 'mysql', '~> 9.0'
+depends 'mysql', '~> 10.0'
```
Comment (ramereth) [inline README.md:51]: It might make more sense to not have the version pinned here so we don't have to keep updating it.
```suggestion
depends 'mysql'
```

**Review by ramereth — COMMENTED** [review]
Can you also remove Ubuntu 16.04 in the readme as we removed it in #655 and was missed?

**Comment by axos88-da** [comment]
Done

**Comment by ramereth** [comment]
@axos88-da can you please add a changelog entry?

**Comment by axos88-da** [comment]
For changing the README.md file?

**Comment by ramereth** [comment]
@axos88-da yeah, we need something in the changelog for our automatic release system to work properly.

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,7 @@
 All notable changes to this project will be documented in this file.
 
 ## Unreleased
```
Comment (ramereth) [inline CHANGELOG.md:5]: This needs to have a newline.

**Review by ramereth — CHANGES_REQUESTED** [review]

**Comment by axos88-da** [comment]
Never mind.

**Comment by kitchen-porter** [comment]
Released as: [10.0.2](https://github.com/sous-chefs/mysql/releases/tag/10.0.2)

**Pull request merged**